content/blog/2024/anchor-position-polyfill-2.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Continued updates to the Anchor Position Polyfill
```

Comment:
Should we standardize on either `Anchor Position Polyfill` or `Anchor Positioning Polyfill`?

Reply:
Anchor Positioning is in the spec, so I'll standardize on that.